appveyor.yml

```diff
@@ -2,8 +2,7 @@ version: 11.8.1.{build}
 image: Visual Studio 2017
 
 environment:
-  STRIPE_TEST_SK:
-    secure: CcMNB0hu30XVzrZiu9MLdpEzejPmX1xqp/kizfUz5s0BaCzAKnfBYRjqO+Q8wTfN
+  STRIPE_TEST_SK: sk_test_eBgAzVoEpJKfYjD9nf2YoyMM
```

Comment:
should we use the bindings account instead if we're to make it public?

Reply:
After giving it a try, it appears the stripe-dotnet test suite expects some specific account settings and cannot run on the general bindings test account.

It's probably not worth trying to fix the tests that fail since we're going to switch the entire test suite to stripe-mock in the mid term.

Reply:
Why are we doing this?